compute_endpoint/globus_compute_endpoint/executors/high_throughput/worker.py

```diff
@@ -180,6 +181,10 @@ def call_user_function(self, message: bytes) -> str:
             task_data = task.task_buffer.decode("utf-8")  # type: ignore[attr-defined]
 
         f, args, kwargs = self.serializer.unpack_and_deserialize(task_data)
+        GC_TASK_TIMEOUT = int(os.environ.get("GC_TASK_TIMEOUT", 0))
+        log.warning(f"GC_TASK_TIMEOUT: {GC_TASK_TIMEOUT}")
```

Comment:
Suggest demoting this to `debug`, or maybe `info` if we only log it when non-zero